sympy/core/evalf.py

```diff
@@ -1375,9 +1388,77 @@ def evalf(x, prec, options):
         check_target(x, r, prec)
     return r
 
+def _evalf_options(x, prec, options):
```

Comment:
This function should just be called `evalf_options` i.e. no underscore because it's not private.